docs/markdown/ios/card/main_component.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Card
+description: The Card is a semi-modular container used to house a Card.Header and a variant of the available preview blocks. Every Card must have at least a title and a variant of a preview block.
```

Comment:
I can also take a separate copyediting pass at this once this PR is merged, if that's easier

Reply:
thanks, for bringing this up! I think this can be fixed easily with a find and replace. These were written by a number of people, some outside of Gestalt, so it's not consistent. I also need clarification on what it means to say "a specific instance." I see this in the style guide, but not always sure what it means.